lib/open_mic.rb

```diff
@@ -0,0 +1,18 @@
+class OpenMic
+  attr_reader :location, :date, :performers
+  def initialize(info)
+    @info = info
+    @location = @info[:location]
+    @date = @info[:date]
+    @performers = []
+  end
+
+  def welcome(user)
+    @performers << user
+  end
+
+  def repeated_jokes?
+    found_jokes = @performers.find_all {|told_joke| told_joke.told == true}
+    found_jokes.count >= 2 == true
```

Comment:
hard-coding numeric values like this is dangerous and isn't quite what we were looking for here; what if we had 8 performers who all knew 10 jokes and 5 performers told 1 to 4 jokes to one other performer, etc.?